libioc/Config/Jail/File/Fstab.py

```diff
@@ -76,10 +76,16 @@ def __setitem__(
         value: typing.Union[str, libioc.Types.AbsolutePath]
     ) -> None:
         """Set an item of the FstabLine."""
-        if (key == "source") or (key == "destination"):
+        _type = None
+        if key == "source":
+            _type = libioc.Types.Path
+        elif key == "destination":
+            _type = libioc.Types.AbsolutePath
+
+        if _type is not None:  # source or destination
             if isinstance(value, str) is True:
-                absolute_path = libioc.Types.AbsolutePath(value)
-            elif isinstance(value, libioc.Types.AbsolutePath) is True:
+                absolute_path = _type(value)
+            elif isinstance(value, _type) is True:
```

Comment:
what does this mean?

Reply:
This means that in case of `source`, either an instance of Path or AbsolutePath is accepted, in case of `destination`, only an AbsolutePath.

Anyhow, this logic can be moved to the Types.py, so that this code is less complex.